### PR TITLE
Create portuguese translation

### PR DESCRIPTION
**Pull request by murilooon**
This is a PR to translate the NEAR Wallet to Portuguese.

### CODE REVIEW

**Comment by VGasparini** [comment]
@murilooon Could I help you on that?

How can we coordinate a async translate process?

**Comment by murilooon** [comment]
You can clone this repository by the [guild fork](https://github.com/NEAR-Portuguese-Community/near-wallet), them checkout to this branch and push the translation commits here. I think it's possible this way, not sure...

File: src/translations/pt.global.json [hunk 1]
```diff
@@ -0,0 +1,1103 @@
...
+        "ADD_ACCESS_KEY": {
+            "success": "${title} está autorizado a usar sua conta.",
+            "error": "Ocorreu um erro ao aprovar esta ação. Por favor, tente novamente!"
+        },
+        "ADD_ACCESS_KEY_SEED_PHRASE": {
+            "success": "A configuração da recuperação da Frase Semente está completa.",
+            "error": "Ocorreu um erro ao configurar a recuperação da frase semente. Por favor, tente novamente!"
+        },
+        "SETUP_RECOVERY_MESSAGE": {
+            "success": "Configuração de recuperação está completo.",
```
Comment (vitorebatista) [inline src/translations/pt.global.json:23]:
```suggestion
            "success": "A configuração de recuperação está completa.",
```

File: src/translations/pt.global.json [hunk 2]
```diff
@@ -0,0 +1,1103 @@
...
+        "VERIFY_TWO_FACTOR": {
+            "success": "Autenticação de dois fatores (2FA) configurada com sucesso!",
+            "error": "Configuração da autenticação de dois fatores (2FA) falhou. Por favor, tente novamente."
+        },
+        "CHECK_IS_NEW": {
+            "success": "Parabéns! ${accountId} está disponível.",
+            "error": "Essa conta já existe. Por favor, tente novamente."
+        },
+        "CREATE_NEW_ACCOUNT": {
+            "success": "Congrats! ${accountId} is available.",
```
Comment (vitorebatista) [inline src/translations/pt.global.json:74]:
```suggestion
            "success": "Parabéns! ${accountId} está disponível.",
```

**Comment by VGasparini** [comment]
Hi there!

On past weeks we (@murilooon @kammradt and me) coordinate a "test drive" translate process using [Crowdin](https://crowdin.com/project/near-wallet)

Was a very positive experience, we've translated around 80% and those string were commited on this branch.

image

Any suggestion or question, please contact me
:)

**Comment by murilooon** [comment]
Just finished the translations. Also, updated with master and english current json file. :wink:

**Comment by Patrick1904** [comment]
> Just finished the translations. Also, updated with master and english current json file. 😉

Major props to everyone that has contributed to this!

@murilooon it seems that tests are failing after the last commit. Can you please take a look?

**Comment by murilooon** [comment]
> > Just finished the translations. Also, updated with master and english current json file. wink
> 
> Major props to everyone that has contributed to this!
> 
> @murilooon it seems that tests are failing after the last commit. Can you please take a look?

All the commits broke the CI. Something about sentry and an ENV. 

image

Maybe because this is a branch created from the forked repo?

**Comment by vgrichina** [comment]
> Maybe because this is a branch created from the forked repo?

It's just because forked repo, ignore error

**Pull request merged**